mythril/analysis/solver.py

```diff
@@ -109,29 +108,42 @@ def get_transaction_sequence(global_state, constraints):
             minimize.append(transaction.call_data.calldatasize)
             minimize.append(transaction.call_value)
 
-            concrete_transactions[tx_id] = tx_template.copy()
-
         else:
             creation_tx_ids.append(tx_id)
 
     model = get_model(tx_constraints, minimize=minimize)
-
+    min_price_dict = {}  # type: Dict[str, int]
     for transaction in transactions:
         tx_id = str(transaction.id)
-
-        concrete_transactions[tx_id]["calldata"] = "0x" + "".join(
+        concrete_transaction = dict()  # type: Dict[str, str]
+        concrete_transaction["input"] = "0x" + "".join(
             [
                 hex(b)[2:] if len(hex(b)) % 2 == 0 else "0" + hex(b)[2:]
                 for b in transaction.call_data.concrete(model)
             ]
         )
-
-        concrete_transactions[tx_id]["call_value"] = (
-            "0x%x"
-            % model.eval(transaction.call_value.raw, model_completion=True).as_long()
-        )
-        concrete_transactions[tx_id]["caller"] = "0x" + (
+        value = model.eval(transaction.call_value.raw, model_completion=True).as_long()
+        concrete_transaction["value"] = "0x%x" % value
+        caller = "0x" + (
             "%x" % model.eval(transaction.caller.raw, model_completion=True).as_long()
         ).zfill(40)
 
-    return concrete_transactions
+        concrete_transaction["origin"] = caller
+        concrete_transaction["address"] = "%s" % transaction.callee_account.address
+        concrete_transactions.append(concrete_transaction)
+        min_price_dict[caller] = min_price_dict.get(caller, 0) + value
+    initial_state = transaction_sequence[0].world_state.accounts
```

Comment:
If the creation transactions are not included, shouldn't we take the world state after the creation and not before?

Reply:
It ran perfectly in my tests, there seems to be a copy stuff not happening somewhere in worldstate.